packages/tpetra/core/src/Tpetra_Details_cublasGemm.cpp

```diff
@@ -56,19 +56,19 @@ namespace Cublas {
 namespace Impl {
 
 void
-cgemm (const char transA,
-       const char transB,
-       const int m,
-       const int n,
-       const int k,
-       const Kokkos::complex<float>& alpha,
-       const Kokkos::complex<float> A[],
-       const int lda,
-       const Kokkos::complex<float> B[],
-       const int ldb,
-       const Kokkos::complex<float>& beta,
-       Kokkos::complex<float> C[],
-       const int ldc)
+cgemm (const char /* transA */,
+       const char /* transB */,
+       const int /* m */,
+       const int /* n */,
+       const int /* k */,
+       const Kokkos::complex<float>& /* alpha */,
+       const Kokkos::complex<float> /* A */[],
```

Comment:
Is this the right way to handle these situations?

Reply:
I don't think you can do this with a simple script.  You can see below that this will break the CUDA build.